src/plugins/ConditionalRule.js

```diff
@@ -18,6 +19,7 @@ export default class ConditionalRule {
 
   constructor(selector: string, styles: Object, options: RuleOptions) {
     this.selector = selector
+    this.selectorId = selector.replace(' ', '')
```

Comment:
but we store it like `@keyframe name`, and `CSSKeyframesRule.name` would be just `name`

you suggest to build `selector` by hands like 
```
`@keyframe ${CSSKeyframesRule.name}`?
```